ProjectLighthouse/Database.cs

```diff
@@ -451,6 +451,12 @@ public bool IsRegistrationTokenValid(string tokenString)
 
     public async Task RemoveExpiredTokens()
     {
+        foreach (GameToken token in this.GameTokens.Where(t => DateTime.Now > t.ExpiresAt).ToList())
+        {
+            User? user = await this.Users.FirstOrDefaultAsync(u => u.UserId == token.UserId);
+            if(user != null) user.LastLogout = TimeHelper.TimestampMillis;
+            this.GameTokens.Remove(token);
+        }
         this.GameTokens.RemoveWhere(t => DateTime.Now > t.ExpiresAt);
```

Comment:
Is this necessary if on the above lines the gametokens are cleared out anyway

Reply:
Good catch, I forgot to get rid of the original line that removed game tokens